packages/plugin-dom-helpers/src/DomHelpers.ts

```diff
@@ -175,6 +175,19 @@ export class DomHelpers<T extends JWPluginConfig = JWPluginConfig> extends JWPlu
         }
         await this.editor.dispatcher.dispatchHooks('@redraw');
     }
+    /**
+     * Prepend a DOM Node inside the child of another.
+     *
+     * @param params
+     */
+    async prepend(fromDomNode: Node, toDomNode: Node): Promise<void> {
+        const toNodes = this.getNodes(toDomNode);
+        const toNode = toNodes[toNodes.length - 1];
+        for (const fromNode of this.getNodes(fromDomNode).reverse()) {
+            fromNode.prepend(toNode);
+        }
+        await this.editor.dispatcher.dispatchHooks('@redraw');
+    }
```

Comment:
So... the implementation is alright by me, but I have questions regarding the concept. We have `moveBefore`. Can't it already be used for that?
Like `moveBefore(fromDomNode, toDomNode.firstChild)` ? If it's specifically for nodes that don't have children yet, then why specifically `prepend` and not `append` ?
In any case, if you make a `prepend`, please also make a `append` :-)

Reply:
The reason is that we do not want to check if there is a child or not.